Packs/Base/Scripts/ValidateContent/ValidateContent.py

```diff
@@ -0,0 +1,384 @@
+    """
+    Create empty 'README.md', '.secrets-ignore', and '.pack-ignore' files that are expected
+    to be in the base directory of a pack
+    """
+    fp = open(os.path.join(self.pack_dir_path, 'README.md'), 'a')
+    fp.close()
+
+    fp = open(os.path.join(self.pack_dir_path, '.secrets-ignore'), 'a')
+    fp.close()
+
+    fp = open(os.path.join(self.pack_dir_path, '.pack-ignore'), 'a')
+    fp.close()
+
+
+def content_item_to_package_format(
+        self, content_item_dir: str, del_unified: bool = True, source_mapping: Optional[Dict] = None  # noqa: F841
+) -> None:
+    child_files = get_child_files(content_item_dir)
+    for child_file in child_files:
+        cf_name_lower = os.path.basename(child_file).lower()
+        if cf_name_lower.startswith((SCRIPT, AUTOMATION, INTEGRATION)) and cf_name_lower.endswith('yml'):
+            content_item_file_path = child_file
+            file_type = find_type(content_item_file_path)
+            file_type = file_type.value if file_type else file_type
+            try:
+                extractor = Extractor(
+                    input=content_item_file_path, file_type=file_type, output=content_item_dir, no_logging=True,
+                    no_pipenv=True)
+                extractor.extract_to_package_format()
+            except Exception as e:
+                err_msg = f'Error occurred while trying to split the unified YAML "{content_item_file_path}" ' \
+                          f'into its component parts.\nError: "{e}"'
+                self.contrib_conversion_errs.append(err_msg)
+            if del_unified:
+                os.remove(content_item_file_path)
+
+
+def format_converted_pack(self) -> None:
+    """Runs the demisto-sdk's format command on the pack converted from the contribution zipfile"""
+    from_version = '6.0.0' if self.create_new else ''
+    format_manager(
+        input=self.pack_dir_path, from_version=from_version, no_validate=True, update_docker=True, assume_yes=True
+    )
+
+
+def convert_contribution_to_pack(contrib_converter: ContributionConverter) -> None:
+    """Create or updates a pack in the content repo from the contents of a contribution zipfile
+
+    Args:
+        contrib_converter (ContributionConverter): Contribution contributor object
+    """
+    # only create pack_metadata.json and base pack files if creating a new pack
+    if contrib_converter.create_new:
+        if contrib_converter.contribution:
+            # create pack metadata file
+            with zipfile.ZipFile(contrib_converter.contribution) as zipped_contrib:
+                with zipped_contrib.open('metadata.json') as metadata_file:
+                    metadata = json.loads(metadata_file.read())
+                    contrib_converter.create_metadata_file(metadata)
+        # create base files
+        contrib_converter.create_pack_base_files = types.MethodType(_create_pack_base_files, contrib_converter)
+        contrib_converter.create_pack_base_files()
```

Comment:
why do you need to override the function? when you can simply call it with `contrib_converter.pack_dir_path`?
what added value do you get here?

Reply:
basically its to run this function without needing to capture output, as in the sdk it runs `click.echo`